src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -260,10 +259,10 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
-    <Content Include="secrets\LICENSE.txt">
+    <!--<Content Include="secrets\LICENSE.txt">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
+    </Content>-->
```

Comment:
There is another section in this file that adds the secrets assemblies is included in the correct place in the VSIX:

```
  <!-- *************************************************************************** -->
  <!-- Sonar Secrets -->
  <!-- *************************************************************************** -->
  <!-- In the VSIX, the secrets binary(ies) should be in the secrets subfolder next to the specific license file.
       We don't need to copy the file(s). It's enough to specify the VSIXSubPath we want.
  -->
  <Target Name="SecretsBinariesToVSIX" AfterTargets="GeneratePkgDef" BeforeTargets="GetVsixSourceItems">
    <ItemGroup>
      <SonarSecretsFilesToAdd Include="$(OutDir)SonarLint.Secrets.*.dll" />
    </ItemGroup>
    
    <Message Importance="high" Text="Adding the SonarSecrets files to the VSIX:@(SonarSecretsFilesToAdd)" />
    
    <ItemGroup>
      <VSIXSourceItem Include="@(SonarSecretsFilesToAdd)">
        <VSIXSubPath>secrets</VSIXSubPath>
      </VSIXSourceItem>
    </ItemGroup>
  </Target>
```

I didn't bother to comment it out because it wasn't necessary - no files in the directory match `SonarLint.Secrets.*.dll`, so nothing gets copied.